.travis.yml

```diff
@@ -20,6 +20,10 @@ jobs:
     env: TARGET=functional-tests TEST_FUNCTIONAL="fedora-generic-tiny fedora-generic-small fedora-generic-medium fedora-generic-large"
     script: bash -x test.sh
 
+  - name: Functional test of the desktop Fedora template
+    env: TARGET=functional-tests TEST_FUNCTIONAL="fedora-generic-desktop-tiny fedora-generic-desktop-small fedora-generic-desktop-medium fedora-generic-desktop-large"
```

Comment:
fedora-desktop-small, the desktop part is a workload specification, we are not adding any new type property